datafusion/substrait/src/logical_plan/consumer.rs

```diff
@@ -412,6 +428,10 @@ pub async fn from_substrait_rel(
                 );
                 let mut names: HashSet<String> = HashSet::new();
                 let mut exprs: Vec<Expr> = vec![];
+                input.schema().iter().for_each(|(qualifier, field)| {
+                    exprs.push(col(Column::from((qualifier, field))))
+                });
```

Comment:
this seems like the right thing - but then to get tests passing again I think we'll also need to add support for the `remap` https://substrait.io/relations/basics/#emit-output-ordering, both here and in producer...